app/models/manageiq/providers/redhat/infra_manager/refresh_parser.rb

```diff
@@ -415,6 +411,20 @@ def self.vm_inv_to_hashes(inv, _storage_inv, storage_uids, cluster_uids, host_ui
     return result, result_uids
   end
 
+  def self.create_vm_hash(data)
```

Comment:
Can you pass these in as named parameters?  It'll be easier to read than just as an array